src/pkg/selectors/groups.go

```diff
@@ -217,9 +217,8 @@ func (s *groups) AllData() []GroupsScope {
 	scopes = append(
 		scopes,
 		makeScope[GroupsScope](GroupsLibraryFolder, Any()),
-		makeScope[GroupsScope](GroupsChannel, Any()))
-	// TODO: enable conversations in all-data backups
-	// makeScope[GroupsScope](GroupsConversation, Any()))
+		makeScope[GroupsScope](GroupsChannel, Any()),
+		makeScope[GroupsScope](GroupsConversation, Any()))
```

Comment:
are we ready to release this for public use?  If we're not completely confident that conversations is ready to go, we shouldn't enable this.

Also, there should be a related CLI change that's missing: adding conversations as a data type to the `--data` flag.  Otherwise we'll be able to backup converstions when backing up all data types, but won't be able to back up conversation alone.

Reply:
>  If we're not completely confident that conversations is ready to go, we shouldn't enable this.

We are already allowing backups under the `feature preview` disclaimer. This PR fixes the issue with conversations details test. I don't see any harm in enabling it. Correct me if wrong.

> Also, there should be a related CLI change that's missing: adding conversations as a data type to the --data flag.

Huh. I was able to do a backup with `--data conversations` on main. 
```
./corso backup create groups --group CorsoCITeam --data conversations

Backing Up ∙ CorsoCITeam
Discovering items to backup                   3s done
  Posts                                       2s done (found 10 conversations)
Backing up data                               0s done
Backup complete ∙ bddc9457-f0f9-4867-a298-98cb7a0ce3d3
Bytes Uploaded: 0 B | Items Uploaded: 0 | Items Skipped: 0 | Errors: 0
```